client/src/declarations.d.ts

```diff
@@ -9,6 +9,26 @@ declare module '*.svg' {
   const content: string;
   export default content;
 }
+declare module '*.woff' {
+  const url: string;
+  export default url;
+}
+declare namespace NodeJS {
+  interface Global {
+    MathJax: {
+      Hub: {
+        Config: (attributes: {
+          tex2jax: {
+            inlineMath: Array<string[]>;
+            processEscapes: boolean;
+            processClass: string;
+          };
+        }) => void;
+        Queue: (attributes: unknown[]) => void;
+      };
+    };
+  }
+}
```

Comment:
Is this not redundant with the `MathJax` declaration on line 44?

Reply:
I thought so, but last time I removed it, Linter complained.